tjdashboard/code/site/helpers/tjdashboard.php

```diff
@@ -0,0 +1,92 @@
+<?php
```

Comment:
Why helper ? This should be in the widget model

Reply:
We can access this from anywhere where we already have widget id.

Reply:
OK